### ISSUE TITLE
version confusion

### ISSUE THREAD

**Issue by philfry** [pulp/pulp_container#1689]
Hi all,

I installed pulp-container using pip from pypi where 2.20.0 is the current version, see https://pypi.org/project/pulp-container/#history and https://pulpproject.org/pulp_container/changes/, even though there have been releases after 2.20.0 with lower version numbers.

Looking at this repo, 2.16.8, which was released last week, seems to be the latest version, and the tags seem out of order:
```
$ git tag -l -n1 | tail -n 10
2.16.6          Release 2.16.6
2.18.1          Release 2.18.1
2.19.0          Release 2.19.0
2.19.1          Release 2.19.1
2.19.2          Release 2.19.2
2.19.3          Release 2.19.3
2.20.0          Release 2.20.0
2.14.14         Release 2.14.14
2.16.7          Release 2.16.7
2.16.8          Release 2.16.8
```
I didn't find any hints in the documentation, maybe a specific minor version is only suitable for a specific pulpcore version?
For pulpcore 3.55.1, there's no release for pulp_container yet, right? What would be the pulp_container version with pulpcore 3.55 compatibility?

**Comment by lubosmj** [comment]
Yes, you are right. We did not release a version compatible with pulpcore v3.55. We will strive to do the release next week (2.21?). The commit adding support for the aforementioned version landed into `main` two days ago: https://github.com/pulp/pulp_container/commit/a832b9fc9a06cf96fda20fc7203f06a2baa6b9c4.

Only specific minor versions are suitable for a specific pulpcore version. We do not have any compatibility matrix available at the moment. Would like to see one in the future? For now, it is needed to look into the requirements in respective release branches (e.g., https://github.com/pulp/pulp_container/blob/2.14/requirements.txt).

**Comment by philfry** [comment]
Thanks for the explanation, lubosmj!
A compatibility matrix (major.minor → pulpcore x.y) would be great, or at least mentioning the exclusive compatibility and giving the hint with the requirements.txt in the documentation.

**Comment by lubosmj** [comment]
We are releasing pulp-container this week.

**Comment by lubosmj** [comment]
I am closing this issue in favour of https://github.com/pulp/pulpcore/issues/5573.
